app/models/custom_button_set.rb

```diff
@@ -92,8 +92,14 @@ def deep_copy(options)
     options.each_with_object(dup) { |(k, v), button_set| button_set.send("#{k}=", v) }.tap do |cbs|
       cbs.guid = SecureRandom.uuid
       cbs.name = "#{name}-#{cbs.guid}"
+      cbs.set_data[:button_order] = []
       cbs.save!
-      custom_buttons.each { |cb| cbs.add_member(cb.copy(:applies_to => options[:owner])) }
+      custom_buttons.each do |cb|
+        cb_copy = cb.copy(:applies_to => options[:owner])
+        cbs.add_member(cb_copy)
+        cbs.set_data[:button_order] << cb_copy.id
```

Comment:
Ok, sorry for the delay, I had to refresh my memory of how button and button set relations worked...

I feel like this line belongs as part of `CustomButtonSet#add_member`, what do you think?  Is there a case where you would want to add the member to the set, but not add it to the member order?

Reply:
Yeah I was wondering about this. It didn't make sense to me to have add_member but not change the order, but it's what we're doing. I'd rather fix it there.

Reply:
We can change it in a separate PR